cloudshell-orch-core/cloudshell/workflow/orchestration/apps_configuration.py

```diff
@@ -9,24 +9,30 @@ def __init__(self, sandbox):
         self.sandbox = sandbox
         """:type : Sandbox"""
 
-    def set_config_param(self, app, key, value):
+    def set_config_param(self, app, script_alias, key, value):
         """
         :param App app:
+        :param script_alias:
         :param str key:
         :param str value:
         :return:
         """
         if isinstance(app, App):
             # If deployed in this execution
             if app.app_request.app_resource is not None:
```

Comment:
I'm not aware yet of what `app_resource` is, do we have to explicitly check for `None`? Or regular truthiness check is ok? @dannykQuali what do you think? If so, this can be simplified to: `if app.app_request.app_resource`

Reply:
Actually I did not modify this line, but I prefer explicit checking rather that vague all-in-one check.
Remember that we are not mainly python developers, so as explicit as the code can be - the better.